config.cmake.template

```diff
@@ -50,13 +50,48 @@
 # set(BDDCML_ROOT "$ENV{HOME}/local/BDDCML/bddcml-2.2")
 
 ### Python setting ###############################################################
-# USE_PYTHON - only use python if this is set to "yes"
+# USE_PYTHON - use embedded python if this is set to "yes"
 # We add python library only on explicit request since it leads to tons of errors when debugging with valgrind.
 #
 # set(USE_PYTHON "yes")
 
 # PYTHON_COPY - make copy of python files, necessary for release builds
 # set(PYTHON_COPY "yes")
+#  - Following variables MUST ALSO BE SET:
+#    - PYTHON_COPY_ROOT
```

Comment:
OK. Leave the name, and explain how it is used. I guess that this path is stripped from other paths and resulting relative path is recreated under the build_tree.